out_writeapi.go

```diff
@@ -425,6 +471,14 @@ func FLBPluginInit(plugin unsafe.Pointer) int {
 
 	configMap[configID] = &config
```

Comment:
How about updating the configMap only after the buildStream call has succeeded? That way in case buildStream fails, we won't modify the configMap.

Reply:
Done.